instagram/graminsta/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```

Comment:
you can remove all the starter styles they added in the boilerplate and set up your own. remember to use resets and utilize the responsive methods we learned like setting our box-sizing: border-box and setting img {width: 100%; height: auto} etc